src/erlfmt.erl

```diff
@@ -36,7 +36,9 @@
 
 -type error_info() :: {file:name_all(), erl_anno:location(), module(), Reason :: any()}.
 -type pragma() :: require | insert | delete | ignore.
--type config() :: [{pragma, pragma()} | {print_width, pos_integer()} | verbose].
+-type config() :: [
+    {pragma, pragma()} | {print_width, pos_integer()} | {filename, string()} | verbose
+].
```

Comment:
The `config()` option is used in more places than just in `format_string`, and it seems in others we don't support the `{filename, _}` option. It might make sense to have a separate type there.

Reply:
It means we should have a config type for format function?

Reply:
Or we just move available config in the list to a new type and union it with filename config at format_string spec

Reply:
The second option is fine with me

Reply:
Fixed. 🙇